lib/oban/validation.ex

```diff
@@ -167,6 +167,16 @@ defmodule Oban.Validation do
     end
   end
 
+  defp validate_type(:mfa, key, {module, func, args})
+       when is_atom(module) and is_atom(func) and is_list(args) do
+    if function_exported?(module, func, length(args)) do
+      :ok
+    else
+      {:error,
+       "missing function #{inspect(Exception.format_mfa(module, func, length(args)))} for #{key}"}
```

Comment:
Exception.format_mfa/3 returns a string, so using `inspect/1` escapes the quotes.

```suggestion
      {:error, "missing function #{Exception.format_mfa(module, func, length(args))} for #{key}"}
```